cpp/pipelines/add_authority_external_ref.cc

```diff
@@ -18,14 +20,56 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <chrono>
 #include <fstream>
 #include <iostream>
 #include <regex>
+#include <string>
+#include <nlohmann/json.hpp>
 #include "BeaconFile.h"
+#include "FileUtil.h"
 #include "MARC.h"
 #include "StringUtil.h"
+#include "TextUtil.h"
 #include "util.h"
 
+/**
+ * ============= Generate gnd_to_wiki ===============
+ * New scenario to generate gnd_to_wiki.txt
+ * by Steven Lolong
+ * update-1: 16.12.2022
+ *
+ * Case of "generate_gnd_to_wiki"
+ * The input_txt_file is the file generated by this command:
+ *      jq -c --stream '.' < authorities-gnd-person_lds.jsonld |grep -E 'https\\:/\\/d-nb\\.info\\/gnd\\/|wikidata|wikipedia'
```

Comment:
As discussed, the comments regarding integration (jq and so on) should be moved into a bash script that actually downloads the latest version and starts the processing including all the jq commands + calling this program.